From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Tue, 21 Jan 2020 11:01:50 +0100
Subject: arm64: bcm2835: Drop select of nonexistent HAVE_ARM_ARCH_TIMER
Git-commit: feefd39ab6bb6a746364ef00c39cce2e33a0fde7
Patch-mainline: v5.7-rc1
References: bsc#1171079

Unlike on arm32, the HAVE_ARM_ARCH_TIMER config symbol does not exist on
arm64.

Note that the toplevel ARM64 symbol always selects ARM_ARCH_TIMER, so
support for it is always included.

Fixes: 628d30d1ccb897fe ("arm64: Add platform selection for BCM2835.")
Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Reviewed-by: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
Signed-off-by: Florian Fainelli <f.fainelli@gmail.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 arch/arm64/Kconfig.platforms |    1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -38,7 +38,6 @@ config ARCH_BCM2835
 	select PINCTRL_BCM2835
 	select ARM_AMBA
 	select ARM_TIMER_SP804
-	select HAVE_ARM_ARCH_TIMER
 	help
 	  This enables support for the Broadcom BCM2837 SoC.
 	  This SoC is used in the Raspberry Pi 3 device.

